application.py

```diff
@@ -13,6 +16,9 @@
     ]
 }
 
+# Sadly can't enable this yet, but we should work towards it...
+# application.jinja_env.undefined = jinja2.StrictUndefined
```

Comment:
What's preventing this?

Reply:
Ah I made a tech debt ticket for this.

Basically our frontend toolkit doesn't use Jinja's `is defined` check, it instead just does `{% if some_var %}` (rather than eg `{% if some_var is defined %}`). Without strict undefined, the first test compiles but evaluates to False. With strict undefined, the first won't compile, so we need to change all of the checks like that to use the defined test.

I think it's worth doing but it's way too big a thing to do for now.